SpatialGDK/Source/SpatialGDK/Private/SpatialView/WorkerView.cpp

```diff
@@ -41,6 +41,8 @@ void WorkerView::SendAddComponent(Worker_EntityId EntityId, ComponentData Data,
 	EntityViewElement* Element = View.Find(EntityId);
 	if (ensure(Element != nullptr))
 	{
+		checkf(!Element->Components.ContainsByPredicate(ComponentIdEquality{ Data.GetComponentId() }),
+			   TEXT("Trying to add duplicated component %d to entity %lld"), Data.GetComponentId(), EntityId);
```

Comment:
Nit - prefer having all the string interpolation at the end. So "Tried to add duplicate component to entity. Entity ID: Component ID:"